docs/setup.md

```diff
@@ -132,42 +121,50 @@ Also ensure you have firefox installed
 2. Install the dependencies:
 
    ```bash
-   bundle install && yarn install
+   rbenv exec bundle install && yarn install
```

Comment:
Why do we need to rbenv exec this? Because we have a `.ruby-version` file rbenv will automatically detect and use the correct version.
<img width="1835" height="281" alt="image" src="https://github.com/user-attachments/assets/50256bc1-6cae-4d5d-b067-e7349b05bf77" />
See me testing this by switching between two codebases with different ruby versions here.